.changeset/fast-onions-drop.md

```diff
@@ -0,0 +1,7 @@
+---
+"sit-onyx": major
+"@sit-onyx/nuxt": major
+---
+
+- feat(OnyxNavItem): OnyxNavItem is universal usable and nestable.
+- feat!(OnyxNavButton): deprecated `OnyxNavButton`, Use `OnyxNavItem` instead.
```

Comment:
Since we are in beta, we can also remove the nav button directly :)
If you decide to just deprecate it, it is not a major change, right?